### PR TITLE
dev-scheme/elk: Patches for implicit-int and ptrs

### PR DESCRIPTION
**Pull request by navi-desu**
Closes: https://bugs.gentoo.org/870475

### CODE REVIEW

**Comment by xgqt** [comment]
Thx navi!

File: dev-scheme/elk/files/elk-3.99.8-implicit-int-incompat-ptr.patch [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
++++ b/lib/xwidgets/xaw/list.c	2023-04-22 14:32:58.923738362 -0300
+@@ -2,14 +2,15 @@
+ #include <X11/Xaw/List.h>
+ 
+ 
+-static char **Get_List (Object x) {
++static const char **Get_List (Object x) {
+     register int i, n;
+-    register char *s, **l;
++    register char *s;
```
Comment (thesamesam) [inline dev-scheme/elk/files/elk-3.99.8-implicit-int-incompat-ptr.patch:42]: Please drop register whenever you see it too. It doesn't do anything and it'll be removed at some point.

**Review by thesamesam — COMMENTED** [review]
Always revbump for modern C fixes, too, as they often affect codegen.

File: dev-scheme/elk/files/elk-3.99.8-implicit-int-incompat-ptr.patch (hunk 1 above)
Comment (navi-desu) [reply dev-scheme/elk/files/elk-3.99.8-implicit-int-incompat-ptr.patch:42]: Noted, about revbumping and register

**Pull request closed without merging**